src/flags.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import {ENV} from '@tensorflow/tfjs-core';
+
+/**
+ * Thread register block size for matmul kernel. If 0, we use the version of
+ * matMul without register blocking.
+ */
+ENV.registerFlag('MATMUL_WORK_PER_THREAD', () => 4);
```

Comment:
I will get error when launching yarn test if I manually change 4 to 0. Can you take a look?

Reply:
The root cause is a typo in matmul.ts, we don't met this before as it always run `new MatMulPackedProgram` instead of `new MatMulProgram`, I fixed this in commit 41807de